scipy/sparse/linalg/tests/test_special_sparse_arrays.py

```diff
@@ -76,10 +76,14 @@ def test_eigenvalues(self, grid_shape, bc):
         eigenvalues = lap.eigenvalues
```

Comment:
Just call `lap.eigenvalues()` where you need to.

```suggestion
```

Reply:
done

Reply:
Still looks unchanged to me. Do you need to push?

Reply:
Actually, I would suggest getting the all-eigenvalues array here and using that array down below where we used to instead of calling `lap.eigenvalues()` each time. Then, in the "test every ``m > 0``" clause that you added, you can call `lap.eigenvalues(m)` repeatedly.

Reply:
Sorry, lost my focus... I'll let the current CI finish first